client/src/components/BottomNav/BottomNav.js

```diff
@@ -39,8 +41,14 @@ const BottomNav = () => {
           <HomeButton to="/">
             <img src={HomeIcon} alt={"Zur Wochenübersicht"} />
           </HomeButton>
-          <IconButton iconSrc={ShareIcon} iconAlt={"ShareIcon"} />
-          <IconButton iconSrc={DeleteIcon} iconAlt={"DeleteIcon"} />
+          <HomeButton to="/Einkaufsliste">
+            <img src={GroceryIcon} alt={"Zur Einkaufsliste"} />
+          </HomeButton>
+          <HomeButton to="/Rezeptübersicht">
+            <img src={RecipeIcon} alt={"Zur Rezeptübersicht"} />
```

Comment:
```suggestion
            <img src={GroceryIcon} alt="Zur Einkaufsliste" />
          </HomeButton>
          <HomeButton to="/Rezeptübersicht">
            <img src={RecipeIcon} alt="Zur Rezeptübersicht" />
```
I don't think you need curly brackets if you type in the alt text as a string.

Reply:
thanks :)